Tests/AssertionDependencyTests/AssertionDependencyTests.swift

```diff
@@ -0,0 +1,28 @@
+import AssertionDependency
+import Dependencies
+import XCTest
+
+final class AssertionDependencyTests: XCTestCase {
```

Comment:
`XCTExpectFailure` doesn't work on Linux, so the (implicit) convention right now is to wrap these tests in `#if os(macOS)` (at least), so they run in CI.